static/scripts/rewards/cirip/query-reverse-ens.ts

```diff
@@ -1,11 +1,11 @@
-import { AppState } from "../app-state";
+import { app } from "../app-state";
```

Comment:
Previously `useRpcHandler()` was being called twice to return a provider on load, once in this file and once in `read-claim-data-from-url.ts`  and `{networkId: networkdID} as AppState` was being used here. Now we take the provider from `app` instead of creating a new one.